test/l4lb-xdp/Vagrantfile

```diff
@@ -0,0 +1,34 @@
+# Vagrant box for testing kind with cgroup v2
```

Comment:
Is Vagrant the only way to accomplish this? I can see some info on this in the PR description, but it would be quite helpful to have in a comment.

Reply:
I've added a comment which hopefully helps to explain why we need this.